databricks-sdk-java/pom.xml

```diff
@@ -93,16 +93,4 @@
       <version>1.20.0</version>
     </dependency>
   </dependencies>
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <configuration>
```

Comment:
why are we removing this?

Reply:
I had added this, but caused failures, so removed